<launch>
  <arg name="namespace" default="ackermann_vehicle"/>
  <arg name="source_package" default="prometheus_gazebo"/>
  <group ns="$(arg namespace)">
    <node pkg="prometheus_gazebo" type="gazebo_signal_panel" name="semaphore_node" output="screen">
      <param name="source_package" value="$(arg source_package)"/>
      <param name="topic_monitor1" value="/ackermann_vehicle/monitor1/image1"/>
      <param name="topic_monitor2" value="/ackermann_vehicle/monitor2/image2"/>
    </node>
  </group>
</launch>
